### PR TITLE
add some awesome speakers of the perl community

### PR DESCRIPTION
**Pull request by eiro**
everyone was contacted by mail, irc, tweeter

### CODE REVIEW

**Comment by SaraVieira** [comment]
I'm sorry , I can't merge because some of the speakers don't have their city :(

File: README.md [hunk 1]
```diff
@@ -347,6 +371,12 @@ Wassim Chegham
 Topics: Angular, JavaScript, Progressive Web Apps, Chatbots, Firebase, Cloud, Internet of Things  
 https://twitter.com/manekinekko
 
+<img src="https://pbs.twimg.com/profile_images/760590433508585472/4062rqec_bigger.jpg" height="70px" width="auto" align="left" alt="" />
+
+Curtis "Ovid" Poe  
+Topics: Perl, Perl6, Relational Model (SQL, ORMs), Good Development Practices
+https://twitter.com/OvidPerl
```
Comment (Ovid) [inline README.md]: I now live in Cannes, France. I'd also add "Agile Training" and "Enterprise Testing" to that list.

**Review by Ovid — CHANGES_REQUESTED** [review]
I'm no longer in Paris :)

**Comment by eiro** [comment]
hello @SaraVieira,

now everyone has a town, please tell me if something is missing.

thanks

**Comment by SaraVieira** [comment]
Hello !

Only saw this now but I'll get it merged :)

Just added a couple of pictures missing and fixed @Ovid  request 🙌

Thank you 🙏

**Pull request merged**